dockers/docker-fpm-frr/daemons

```diff
@@ -36,7 +36,7 @@ fabricd=no
 # Check /etc/pam.d/frr if you intend to use "vtysh"!
 #
 vtysh_enable=yes
-zebra_options="  -A 127.0.0.1 -s 90000000 -M fpm"
```

Comment:
Why are you removing this?

Reply:
That actually should not have happened. I can't recall why/how that file would have changed, or perhaps I merged something in which I shouldn't have. My bad!

Thank you and good catch. It's been fixed.